packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -180,11 +180,12 @@ const Bulletin = ({
   isLive,
   liveText,
   offScreenText,
-  lang,
 }) => {
   const isAudio = mediaType === 'audio';
   const bulletinType = isAudio ? 'radio' : 'tv';
   const BulletinWrapper = isAudio ? RadioBulletinWrapper : TVBulletinWrapper;
+  const ctaTextIsEnglish = ctaText === 'Watch' || ctaText === 'Listen';
```

Comment:
Looks good, but I was thinking to do this in Simorgh instead, as we do in the `StoryPromo` container with the Live label https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/index.jsx#L101. Happy to hear other people thoughts

Reply:
Good idea.
I think that we'll be a better approach to separate the psammead components from our simorgh implementation.